docs/en/TestPayments.md

```diff
@@ -0,0 +1,70 @@
+# Testing the payment process
+
+Before putting a site live, it is wise to test payment gateways first (to ensure the payment process behaves as expected).
+You can do this in two different ways:
+
+1. Using the `omnipay/dummy` gateway.
+2. Using `testMode: true` for your selected payment gateway.
+
+## The `omnipay/dummy` gateway
+
+**NOTE: This module is for testing only, do not use on a live site**
+
+[Omnipay dummy](https://github.com/thephpleague/omnipay-dummy) is a demo payment adaptor for omnipay, allowing you to test the
+payment process without signing up for an existing payment provider. This payment provider generates a payment form with standard
+card fields and will either complete or fail an order based on the provided card details.
+
+To start using `omnipay/dummy` ensure you add it via composer:
+
+    composer require omnipay/dummy:^2.0
```

Comment:
I'd suggest to use `composer require --dev omnipay/dummy:^2.0`. Otherwise the dummy gateway will also get installed when you deploy the site.